diff --git a/CMakeLists.txt b/CMakeLists.txt
index 271c5d8..edafd17 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -17,8 +17,8 @@ cmake_policy(SET CMP0007 NEW)
 cmake_policy(SET CMP0012 NEW)
 
 if (${CMAKE_MAJOR_VERSION} GREATER 2)
-    # old policy do not use MACOSX_RPATH
-    cmake_policy(SET CMP0042 OLD)
+    # old policy do not use MACOSX_RPATH - NEW required for newer CMake
+    # cmake_policy(SET CMP0042 OLD)
 endif()
 
 set(MAJOR_VERSION 1)
